src/android/CHIPTool/app/src/main/java/com/google/chip/chiptool/commissioner/CommissioningFragment.java

```diff
@@ -59,7 +66,12 @@ public View onCreateView(
   public void onViewCreated(@NonNull View view, Bundle savedInstanceState) {
     super.onViewCreated(view, savedInstanceState);
 
-    ProgressBar progressBar = getActivity().findViewById(R.id.commissioning_progress);
+    cancelButton = getActivity().findViewById(R.id.cancel_button);
```

Comment:
Use 'view' instead of 'getActivity()' in lines 69 - 74, to call 'findViewById(..)' on, since the layout inflated in onCreateView(..) contains the cancel_button, done_button_done_image, etc. and is returned as the 'view' param in onViewCreated(..)

Reply:
I missed this. Updated.